packages/wasm/crate/src/metadata.rs

```diff
@@ -61,3 +66,117 @@ fn shorten_id(captures: &regex::Captures) -> WasmResult<String> {
         .take(SHORTENED_ID_LENGTH)
         .collect())
 }
+
+#[cfg(test)]
+mod test_helpers {
```

Comment:
Amazing, tests for our Rust code! 🤩

Reply:
Could you move this to the tests folder?

Reply:
Talked with Gabe on Discord — per the [Rust book](https://doc.rust-lang.org/book/ch11-03-test-organization.html#the-tests-directory), we'll just use the `tests` directory for integration tests.